src/emulator/module/src/module.cpp

```diff
@@ -20,6 +20,7 @@
 #include <iostream>
 #include <mutex>
 #include <set>
+#include "util/log.h"
```

Comment:
Could this include be moved up with the other project includes and changed to use angle brackets?

Reply:
why yes it can